packages/api/src/graphql/collection-index.ts

```diff
@@ -1,5 +1,10 @@
-import pkg, { ApolloClient } from "@apollo/client";
-import { createMutateFunction, createQueryFunction } from "./common.js";
+import * as pkg from "@apollo/client";
```

Comment:
You can just 
```ts
import { gql } from "@apollo/client";
```
instead of 
```ts
import * as pkg from "@apollo/client";
const { gql } = pkg;
```

It actually look cleaner and reduce bundle size cost

Reply:
Unable to import this way, skip